src/libs/RequestThrottle.ts

```diff
@@ -1,28 +1,25 @@
-import _ from 'underscore';
 import CONST from '../CONST';
 
 let requestWaitTime = 0;
 
-function clear() {
+function clear(): void {
     requestWaitTime = 0;
 }
 
-/**
- * @returns {Number} time to wait in ms
- */
+function getRandomInt(min: number, max: number): number {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
 function getRequestWaitTime() {
```

Comment:
missing return type?

Reply:
I removed that `getRandomInt` and got one from utils as recommended [here](https://github.com/Expensify/App/pull/26808#discussion_r1321242921), thanks though ;)